### ISSUE TITLE
Official Event Store image needed

### ISSUE THREAD

**Issue by riccardone** [EventStore/eventstore-docker#60]
We have a [PR open on the Docker official-images](https://github.com/docker-library/official-images/pull/5327) repo in order to recognise our Docker Image as official. Once that this is done we will remove the "Note: This container is for development purposes only"

[Some changes to the current image are required](https://github.com/docker-library/official-images/pull/5327) by the maintainers of the official repo and a PR implementing the changes is required

This has nothing to do with the Helm Chart and Kubernetes. We now have an official [Event Store Helm Chart](https://github.com/EventStore/EventStore.Charts)

I close #34 in favour of this new issue

**Comment by arwinneil** [comment]
This issue has been closed due to management of Docker builds moving to [EventStore/EventStore](https://github.com/EventStore/EventStore), and this repository is due to be archived.
